homeassistant/components/sense/sensor.py

```diff
@@ -55,7 +62,16 @@ def __init__(self, name, sensor_type):
 }
 
 # Production/consumption variants
-SENSOR_VARIANTS = [PRODUCTION_ID, CONSUMPTION_ID]
+SENSOR_VARIANTS = [(PRODUCTION_ID, PRODUCTION_NAME), (CONSUMPTION_ID, CONSUMPTION_NAME)]
+
+# Trend production/consumption variants
+TREND_SENSOR_VARIANTS = SENSOR_VARIANTS + [
```

Comment:
Sense should move to using `SensorEntityDescription`, it will make it more clear what sensors are being defined. [Example](https://github.com/home-assistant/core/pull/54846/files#diff-cf5109edebb2f7b936709887b718df72730a593027d49aef6c399c033439f8bbR39)

Another benefit is that you will be able to implement `entity_registry_enabled_default=False` to avoid having all these new entities show up.

Reply:
Hello. Long time user, first time contributor.

I agree that `SensorEntityDescription`s are a great idea, I'm just wondering if that work should be part of this PR. 

The more I look into the concept, and look at how it's been done in other integrations, it seems like it implementing it will increase the scope of the PR. Plus, as I said, I'm new at this and it's going to take longer than it would a more seasoned contributor to do this work.

Reply:
Ok to do later.